idflakies-maven-plugin/src/main/java/edu/illinois/cs/dt/tools/plugin/AbstractIDFlakiesMojo.java

```diff
@@ -0,0 +1,59 @@
+package edu.illinois.cs.dt.tools.plugin;
+
+import edu.illinois.cs.testrunner.configuration.ConfigProps;
+import edu.illinois.cs.testrunner.configuration.Configuration;
+
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugins.annotations.Execute;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProject;
+
+import java.io.File;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.ArrayList;
+import java.util.List;
+
+@Execute(phase = LifecyclePhase.TEST_COMPILE)
+public abstract class AbstractIDFlakiesMojo extends AbstractMojo {
+
+    // Generic properties
+    @Parameter(property = "project")
+    protected MavenProject mavenProject;
+
+    private String pluginCp;
+
+    @Override
+    public void execute() {
+        setDefaults(Configuration.config());
```

Comment:
Double-check whether doing this helps, considering that it seems a general trend is that we are aiming to not use the Configuration thing from testrunner.

Reply:
Same issue as above for the comment on DetectorUtil.java